pages/documentation/developer/api_guide/single-line-diagram_svg-writing.md

```diff
@@ -0,0 +1,214 @@
+We are showing in this guide how to create some single line diagrams
+- first from a test network,
+- then from a CGMES file.
+
+*Note: the 1.8.0-SNAPSHOT version of powsybl-single-line-diagram was used to generate the SVGs of this guide. Because of that, you might experience some differences with the examples shown, which will be corrected by 1.8.0 release (expected: 2020-11-26).*
+
+
+## Prerequisites
+
+### Maven dependencies
+First of all, we need to add some Maven dependencies in our `pom.xml` file:
+
+```xml
+<dependencies>
+    <dependency>
+        <groupId>com.powsybl</groupId>
+        <artifactId>powsybl-single-line-diagram-core</artifactId>
+        <version>${powsybl.sld.version}</version>
+    </dependency>
+    <dependency>
+        <groupId>com.powsybl</groupId>
+        <artifactId>powsybl-iidm-impl</artifactId>
+        <version>${powsybl.core.version}</version>
+    </dependency>
+    <dependency>
+        <groupId>com.powsybl</groupId>
+        <artifactId>powsybl-iidm-test</artifactId>
+        <version>${powsybl.core.version}</version>
+    </dependency>
+    <dependency>
+        <groupId>com.powsybl</groupId>
+        <artifactId>powsybl-config-test</artifactId>
+        <version>${powsybl.core.version}</version>
+    </dependency>
+    <dependency>
+        <groupId>com.powsybl</groupId>
+        <artifactId>powsybl-cgmes-conversion</artifactId>
+        <version>${powsybl.core.version}</version>
+    </dependency>
+    <dependency>
+        <groupId>com.powsybl</groupId>
+        <artifactId>powsybl-triple-store-impl-rdf4j</artifactId>
+        <version>${powsybl.core.version}</version>
+    </dependency>
+    <dependency>
+        <groupId>org.slf4j</groupId>
+        <artifactId>slf4j-simple</artifactId>
+        <version>${slf4j.version}</version>
+    </dependency>
+</dependencies>
+
+<properties>
+    <powsybl.sld.version>1.7.2</powsybl.sld.version>
+    <powsybl.core.version>3.7.1</powsybl.core.version>
+    <slf4j.version>1.7.22</slf4j.version>
+</properties>
+```
+
+Here are some details about these dependencies:
+- `powsybl-single-line-diagram-core` is the core module of single-line-diagram,
+- `powsybl-iidm-impl` is for the Network model,
+- `powsybl-iidm-test` is for loading the test Network,
+- `powsybl-config-test` is for loading the test configuration (see [configuration API guide](configuration.md)),
+- `powsybl-cgmes-conversion` and `powsybl-triple-store-impl-rdf4j`  are for importing a CGMES file,
+- `slf4j-simple` allows you to have simple logging capabilities.
+
+### Layout parameters
+For both example networks, we need to initialize a few layout parameters before generating diagrams (see [SVG layouts]() API guide for more info):
+
+```java
+// Instantiating the default style component library
+ComponentLibrary componentLibrary = new ResourcesComponentLibrary("/ConvergenceLibrary");
```

Comment:
Could you also create an issue to rename Convergence... I don't understand why the Convergence word still appear

Reply:
I'm not sure you saw [my comment above](https://github.com/powsybl/powsybl.github.io/pull/161#discussion_r516601965), as you don't mention it... If you disagree on what's written then I'll create an issue to discuss about it.

Reply:
[PR](https://github.com/powsybl/powsybl-single-line-diagram/pull/203) created on powsybl-single-line-diagram to have a default constructor

Reply:
Hiding Convergence word by creating a default constructor is not really a good option it just hard code Convergence in the default constructor...
I do not see any problem with this name, the fact is it is Convergence style components. And I would not have any problem too if someone wants to add a "PowerFactory" library that looks like PowerFactory style....

Reply:
PowerFactory is a well known software, but Convergence is not. As non RTE or Coreso user, I don't understand what Convergence is, and why the default theme is called Convergence. This is my point